feature-group-definitions/scrollintoview.yml

```diff
@@ -1,3 +1,4 @@
+name: "scrollIntoView()"
```

Comment:
If we wanted to give a context clue here…

```suggestion
name: "scrollIntoView() DOM method"
```

Reply:
More in line with suggestions around Array might be:

```suggestion
name: "Element scrollIntoView() method"
```

But I wonder if we shouldn't save the more uniform names for a longer name field?

Reply:
Yes, a longer name field is better for this. A StackOverflow search makes "scrollIntoView()" alone look sufficient to me.